packages/config/config/devices/0x027a/zen14.json

```diff
@@ -57,118 +57,38 @@
 		},
 		{
 			"#": "2",
-			"label": "Auto-off Timer Outlet 1",
-			"description": "Minutes",
-			"valueSize": 4,
-			"unit": "minutes",
-			"minValue": 0,
-			"maxValue": 65535,
-			"defaultValue": 0,
-			"unsigned": true
+			"$import": "templates/zooz_template.json#auto_off_timer_duration",
+			"label": "Auto Turn-Off Timer Duration (Outlet 1)"
 		},
 		{
 			"#": "3",
-			"label": "Auto-off Timer Outlet 2",
-			"description": "Minutes",
-			"valueSize": 4,
-			"unit": "minutes",
-			"minValue": 0,
-			"maxValue": 65535,
-			"defaultValue": 0,
-			"unsigned": true
+			"$import": "templates/zooz_template.json#auto_off_timer_duration",
+			"label": "Auto Turn-Off Timer Duration (Outlet 2)"
 		},
 		{
 			"#": "4",
-			"label": "Auto-on Timer Outlet 1",
-			"description": "Minutes",
-			"valueSize": 4,
-			"unit": "minutes",
-			"minValue": 0,
-			"maxValue": 65535,
-			"defaultValue": 0,
-			"unsigned": true
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Auto Turn-On Timer (Outlet 1)"
 		},
 		{
 			"#": "5",
-			"label": "Auto-on Timer Outlet 2",
-			"description": "Minutes",
-			"valueSize": 4,
-			"unit": "minutes",
-			"minValue": 0,
-			"maxValue": 65535,
-			"defaultValue": 0,
-			"unsigned": true
+			"$import": "~/templates/master_template.json#base_enable_disable",
+			"label": "Auto Turn-On Timer (Outlet 2)"
 		},
 		{
 			"#": "6",
-			"label": "On/Off Status Recovery After Power Failure",
-			"description": "Choose the recovery state for your double outdoor plug if a power outage occurs",
-			"valueSize": 1,
-			"minValue": 0,
-			"maxValue": 2,
-			"defaultValue": 2,
-			"unsigned": true,
-			"allowManualEntry": false,
-			"options": [
-				{
-					"label": "Both outlets on the plug automatically turn off once power is restored (it ignores the status prior to power outage)",
-					"value": 0
-				},
-				{
-					"label": "Both outlets on the plug automatically turn on once power is restored (it ignores the status prior to the power outage)",
-					"value": 1
-				},
-				{
-					"label": "Both outlets on the plug remember and restore the status prior to the power outage",
-					"value": 2
-				}
-			]
+			"$import": "~/templates/master_template.json#state_after_power_failure_prev_on_off"
```

Comment:
This should be the _off_on_prev variant, no?

Reply:
Confirmed.  Correct.